src/lib/RepStatus.js

```diff
@@ -80,6 +83,8 @@ class RepStatus extends EventEmitter {
             return Promise.resolve();
         }
 
+        // Using content.url_template to guarantee uniqueness
+        Timer.start(`${LOAD_METRIC.convertTime}_${this.representation.content.url_template}`);
```

Comment:
This is going to get started multiple times since when we hit `pending` as a status, we call this function again.

Reply:
Also, we don't really need to key on the url template right? We just care about 'How long did this file spend converting'. Doesn't matter which representation (or does it?)

Reply:
I've made it so start only starts the timer once. I'm now extracting the file id from the template URL, for a similar tag name to the rest